### PR TITLE
0.9.4-prerelease2

### PR DESCRIPTION
**Pull request by Scarjit**
This pr tracks the progress to 0.9.4-prerelease2

### CODE REVIEW

**Comment by Scarjit** [comment]
Testing is blocked, due to my being unable to find the k3os usb stick :(

**Comment by Scarjit** [comment]
Checks:

 - [X] Installs without errors
 - [X] All default services start
 - [ ] Enabling barcode reader works
 - [X] Enabling datainput works
 - [X] Enabling barcodereader works
 - [X] Enabling cameraconnect works

File: docs/static/examples/development.yaml [hunk 1]
```diff
@@ -487,7 +1106,7 @@ write_files:
       echo "Created namespace!"
 
       echo "Install Helm chart united-manufacturing-hub..."
-      if ! helm install united-manufacturing-hub united-manufacturing-hub/united-manufacturing-hub --values "/home/rancher/configs/united-manufacturing-hub-development.yaml" --set serialNumber=$(hostname) --kubeconfig /etc/rancher/k3s/k3s.yaml -n united-manufacturing-hub ; then
+      if ! helm install united-manufacturing-hub united-manufacturing-hub/united-manufacturing-hub --values "/home/rancher/configs/united-manufacturing-hub-development.yaml" --set serialNumber=$(hostname) --kubeconfig /etc/rancher/k3s/k3s.yaml -n united-manufacturing-hub --devel; then
```
Comment (JeremyTheocharis) [inline docs/static/examples/development.yaml:1109]: here also

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: docs/static/examples/development.yaml [hunk 2]
```diff
@@ -29,15 +29,15 @@ write_files:
...
+          enabled: true
           iprange: "192.168.10.1/24"  # IP Range in CIDR notation
           enableKafka: true
           enableMQTT: false
     
         # Warning: unstable
         # cameraconnect searches for GenICam compatible cameras and reads out the pictures
         cameraconnect:
-          enabled: false
+          enabled: true
```
Comment (JeremyTheocharis) [inline docs/static/examples/development.yaml:40]: as discussed

File: docs/static/examples/development.yaml [hunk 3]
```diff
@@ -300,7 +929,7 @@ write_files:
       # factoryinput, grafanaproxy
       # datainput is the infrastructure to allow data entry via Grafana (contains factoryinput, grafanaproxy)
       datainput:
-        enabled: false
+        enabled: true
```
Comment (JeremyTheocharis) [inline docs/static/examples/development.yaml:932]: as discussed

File: docs/static/examples/development.yaml [hunk 4]
```diff
@@ -20,7 +20,7 @@ write_files:
     
         # barcodereader reads out connected USB barcodereaders emulating a keyboard
         barcodereader:
-          enabled: false
+          enabled: true
```
Comment (JeremyTheocharis) [inline docs/static/examples/development.yaml:23]: as discussed

File: docs/static/examples/development.yaml [hunk 5]
```diff
@@ -54,7 +54,7 @@ write_files:
       dataprocessing:
         nodered:
           enabled: true
-          defaultFlows: false
+          defaultFlows: true
```
Comment (JeremyTheocharis) [inline docs/static/examples/development.yaml:57]: as discussed

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

**Review by JeremyTheocharis — APPROVED** [review]

**Comment by Scarjit** [comment]
@JeremyTheocharis requires manual merge, kafka dev private keys kill gitguardian

**Pull request merged**